docs/en/source/quickstart/install.md

```diff
@@ -1,3 +1,86 @@
-# Installation
+# Install TinyMS
 
-TODO
+## User install TinyMS
+
+### Pypi
+
+For users owning a clean environment, recommend using [pypi](https://pypi.org/) to install TinyMS after satisfy the following prerequisites. For setting up python environment, [Anaconda](https://www.anaconda.com/products/individual#Downloads) is a good choice.
```

Comment:
For users who own a clean environment, it is recommend to use pypi to install TinyMS given that the following requirements are meet